great_expectations/exceptions/exceptions.py

```diff
@@ -445,4 +445,5 @@ class DatabaseConnectionError(GreatExpectationsError):
 
 class MigrationError(GreatExpectationsError):
     """Error when using the migration tool."""
+
     pass
```

Comment:
If you have a docstring I don't believe you need the `pass` statement at all. It's valid python.